cmd/cmd.go

```diff
@@ -42,3 +47,10 @@ func (s *HttpCommand) Close() error {
 	}
 	return nil
 }
+
+func commandFiller(ctx *cmd.Context) *form.IOFiller {
```

Comment:
commandFiller sounds like it's filling in a command...
perhaps newIOFiller(*cmd.Context) ?